### PR TITLE
Update building-msvc.md

### PR DESCRIPTION
**Pull request by Mehrshad3**
I added [the URL](https://visualstudio.microsoft.com/downloads/) to download Microsoft Visual Studio 2019.
I'm not **totally sure** that [this URL](https://visualstudio.microsoft.com/downloads/) is True. so if this is true, please accept my pull request to help others download the original Microsoft visual studio. Else, please add the original URL to your doc
thanks

### CODE REVIEW

**Comment by Mehrshad3** [comment]
Why CLA doesn't have parts (b) and (c):exclamation::question:

File: docs/building-msvc.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 Choose an empty folder for the future build, for example **D:\\TBuild**. It will be named ***BuildPath*** in the rest of this document. Create two folders there, ***BuildPath*\\ThirdParty** and ***BuildPath*\\Libraries**.
 
+#### About Visual Stodio 2019
```
Comment (aminya) [inline docs/building-msvc.md:13]:
```suggestion
#### Install Visual Stodio 2019
```

File: docs/building-msvc.md [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+
+* Download **Visual Stodio 2019** installer from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/) and install it.
```
Comment (aminya) [inline docs/building-msvc.md:15]:
```suggestion
* Download the **Visual Studio 2019** installer from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/) and install it.
```

Comment (aminya) [reply docs/building-msvc.md:15]: Probably the "MSCV build tools" is enough to build Telegram. Not sure, the full IDE is a requirement.

Comment (Mehrshad3) [reply docs/building-msvc.md:15]: thanks
I think your suggestion is better

**Comment by john-preston** [comment]
I don't really think this is necessary. The instructions are for building in Visual Studio 2019, and this link is the first one you get when you type "Visual Studio 2019" in Google.

**Comment by Mehrshad3** [comment]
> I don't really think this is necessary. The instructions are for building in Visual Studio 2019, and this link is the first one you get when you type "Visual Studio 2019" in Google.

I had searched and installed a wrong software😬

**Pull request closed without merging**